### PR TITLE
Change the warning about outdated version to mkdocs-material native

### PR DESCRIPTION
**Pull request by oprypin**
https://squidfunk.github.io/mkdocs-material/setup/setting-up-versioning/#version-warning

image

### CODE REVIEW

File: scripts/docs-versions.sh [hunk 1]
```diff
@@ -7,6 +7,6 @@ for ver in $(git ls-remote --heads "${1:-.}" | grep -P -o '(?<=refs/heads/)[0-9]
   printf '{"version": "%s", "title": "%s", "aliases": [%s]}, ' "$ver" "$ver" "$latest"
   latest=''
 done
-printf '{"version": "master", "title": "nightly", "aliases": []}'
+printf '{"version": "master", "title": "nightly", "aliases": ["latest"]}'
```
Comment (straight-shoota) [inline scripts/docs-versions.sh:10]: Why is master latest?

Comment (oprypin) [reply scripts/docs-versions.sh:10]: To not show that it's "(outdated)"

Comment (straight-shoota) [reply scripts/docs-versions.sh:10]: But it's also not the latest stable release. There *should* be a warning that you're reading unreleased documentation.
The message wording "You're not viewing the latest version" fits well for that. It fits for every version that is not the latest stable release.

Comment (oprypin) [reply scripts/docs-versions.sh]: OK

**Review by straight-shoota — APPROVED** [review]

**Pull request merged**